src/aooptionsdialog.cpp

```diff
@@ -129,6 +128,22 @@ AOOptionsDialog::AOOptionsDialog(QWidget *parent, AOApplication *p_ao_app)
   connect(ui_theme_reload_button, &QPushButton::clicked, this,
           &AOOptionsDialog::on_reload_theme_clicked);
 
+  row += 1;
+  ui_theme_folder_button = new QPushButton(ui_form_layout_widget);
+  ui_theme_folder_button->setText(tr("Open Themes Folder"));
+  ui_theme_folder_button->setToolTip(
+      tr("Open the folder where all the themes are contained."));
+  ui_gameplay_form->setWidget(row, QFormLayout::FieldRole, ui_theme_folder_button);
+  connect(ui_theme_folder_button, &QPushButton::clicked, this,
+          [=] {
+    QString p_path = ao_app->get_real_path(VPath("themes/"));
+    if (!dir_exists(p_path)) {
+      return;
+}
+    QDesktopServices::openUrl(QUrl::fromLocalFile(p_path));
+  }
+  );
```

Comment:
If you have multiple paths in assets and a some empty theme folder it will prioritize that one over the others.

It should simply open either the default one (e.g. where the client binary is located) or all themes folder.

![image](https://user-images.githubusercontent.com/26681464/181073607-fdcb330c-77ea-403a-9413-dfc708b56f16.png)
![image](https://user-images.githubusercontent.com/26681464/181073780-14186f18-8667-4ff6-94da-2da2a633026b.png)